app/views/components/_contents-list.html.erb

```diff
@@ -11,7 +12,13 @@
                               end
 %>
 <% if contents.any? %>
-  <nav role="navigation" class="app-c-contents-list">
+  <nav
+    role="navigation"
+    class="app-c-contents-list"
+    <% if aria_label.present? ? %>
```

Comment:
Stray `?`

Reply:
This could be written as 

aria_label ||= false

<%= "aria-label=#{aria_label}" if aria_label %>

Reply:
Fixed. (Copy paste error :(  )